src/content/app/tools/blast/views/blast-submission-results/BlastSubmissionResults.tsx

```diff
@@ -131,20 +146,20 @@ const SequenceBox = (props: SequenceBoxProps) => {
           ></ShowHide>
         </div>
       </div>
-
       {isExpanded &&
         blastResults.map((result) => {
           // TODO: Do we need to show a message if there isn't any matching species? Or will this even ever happen?
-          const speciesInfo = species.find(
+          const speciesInfo = species.filter(
             (sp) => sp.genome_id === result.genomeId
-          ) as Species;
+          );
```

Comment:
This is undoing what the previous PR did to change `filter` to `find`. Perhaps this happened when resolving the conflict?